multiuser/api/che-multiuser-api-resource/src/main/java/org/eclipse/che/multiuser/resource/api/workspace/LimitsCheckingWorkspaceManager.java

```diff
@@ -165,13 +169,34 @@ public WorkspaceImpl updateWorkspace(String id, Workspace update)
   }
 
   @VisibleForTesting
-  void checkMaxEnvironmentRam(WorkspaceConfig config) throws ServerException {
+  void checkMaxEnvironmentRam(Workspace workspace) throws ServerException {
+    if (workspace.getConfig() != null) {
+      checkMaxEnvironmentRam(workspace.getId(), workspace.getConfig());
+    } else {
+      LOG.warn(
+          "Can not estimate memory needed for devfile based workspace `{}`."
```

Comment:
`Can not` -> `Cannot` - that really is the correct spelling

Reply:
I thought I copied/pasted your suggestion :man_facepalming: 
will fix here and recheck all others.